api/v1alpha1/openapi.yaml

```diff
@@ -4192,15 +4198,24 @@ components:
         approved:
           type: boolean
           description: Indicates whether the request has been approved.
```

Comment:
Do we still need the `approved` field, considering that we have an `Approved` condition?

Reply:
hmm..I guess we don't need it anymore. Do we want to remove the `labels` field from the too `EnrollmentRequestApprovalStatus` too? What value does it bring to keep the labels that the ER was approved with ?

Reply:
In the UI, we're currently filtering the list of Pending ER by querying those that have their `status.approval.approved` field either unset or false. So we still need this field, as it's not possible to filter by conditions IIUC.

Reply:
Sorry, I was confused by the comment regarding the Condition, I thought we were talking about the ER status field and not to the response of approving an ER.  
In the response for approving an ER, it's OK to remove the `approved` field.

Reply:
I believe we are talking about the ER status.approved field :)